modules/administration/pages/monitoring.adoc

```diff
@@ -315,6 +315,16 @@ Monitoring formulas can also be configured for System Groups, by applying the sa
 For more information about the monitoring formulas, see xref:specialized-guides:salt/salt-formula-monitoring.adoc[].
 
 
+=== Change Grafana Password
+
+To change Grafana password please follow the steps described in upstream documentation.
+https://grafana.com/docs/grafana/latest/administration/user-management/user-preferences/#change-your-grafana-password
+
+In case you have lost admin password you can reset it using following command:
```

Comment:
```suggestion
In case you have lost the Grafana password you can reset it as [literal]``rooot``using the following command:
```

Reply:
Here the Grafana administrator password is meant.

Reply:
ok, I edited my suggestion.